pkg/daemon/ceph/osd/volume.go

```diff
@@ -459,6 +460,19 @@ func isSafeToUseRawMode(device *DeviceOsdIDEntry, cephVersion cephver.CephVersio
 	return true
 }
 
+func lvmModeAllowed(device *DeviceOsdIDEntry, storeConfig *config.StoreConfig) bool {
+	if device.DeviceInfo.Type == sys.LVMType {
+		logger.Infof("%q is not picked since LVM logical volume doesn't support `metadataDevice` and/or `osdsPerDevice` > 1", device.Config.Name)
+		return false
+	}
+	if device.DeviceInfo.Type == sys.PartType && storeConfig.EncryptedDevice {
+		logger.Infof("%q is not picked since partition doesn't support `metadataDevice` and/or `osdsPerDevice` > 1 if `encryptedDevice` is true", device.Config.Name)
```

Comment:
```suggestion
		logger.Infof("skipping partition %q for lvm mode since encryption is not supported on partitions with a `metadataDevice` or `osdsPerDevice > 1`", device.Config.Name)
```